Sources/JobsPostgres/Migrations/CreateJobQueuedAt.swift

```diff
@@ -0,0 +1,38 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Hummingbird server framework project
+//
+// Copyright (c) 2024 the Hummingbird authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See hummingbird/CONTRIBUTORS.txt for the list of Hummingbird authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import HummingbirdPostgres
+import Logging
+import PostgresNIO
+
+struct CreateJobQueuedAt: PostgresMigration {
+    func apply(connection: PostgresConnection, logger: Logger) async throws {
+        try await connection.query(
+            """
+            ALTER TABLE _hb_pg_jobs ADD COLUMN queued_at TIMESTAMP WITH TIME ZONE NOT NULL
```

Comment:
We need to add a default value for all queued jobs (that already exist). I guess the current date will do